### DISCUSSION TITLE
Microsoft Fluent Design System

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#574] [Discussão]
Deram uma olhada nisso aqui?
http://gizmodo.uol.com.br/microsoft-fluent-desing-system/

Quero a opinião de vocês, principalmente da galera que e voltada para ux, mas também quem foca muito em JS e tem uma pegada mais pra back, o que vocês acham que vai mudar na forma que concebemos as telas,projetos,apps,sites daqui pra frente, com o começo da expansão dos VRS e também essa proposta da Microsoft de mixar realidades? Acha que nosso trabalho vai ficar cada vez mais complexo e interessante? Ou que vao surgir novas profissões, ou especializações ?

**Comment by leandroruel** [comment]
eu vi e achei super legal, mas imagino que algumas coisas dele serão voltadas para 3D.

**Comment by rubenmarcus** [comment]
sim mas vc acha q tudo sera 3d agora? vc acha q uns anos p frente aqui teremos, apps em realidade mista e realidade virtual? Digo,nao so ter apps experimentais, mas ser um padrao do mercado mesmo

**Comment by deppbrazil** [comment]
@rubenmarcus muito show esse vídeo mano, acho que o caminho é mesmo esse mix de realidades  trazendo uma experiência única ao usuário, nos próximos anos teremos muitas profissões que acredito que muitas delas nem podemos imaginar que possa existir, como editor de vídeos holográficos, programador de interfaces híbrida, 3dUX, 3dUI, Designer de tela flexível, enfim e por ai vai...

## :) ##
